includedBuild/gradleConfiguration/src/main/kotlin/com/badoo/reaktive/configuration/MppConfigurationPlugin.kt

```diff
@@ -80,8 +84,23 @@ class MppConfigurationPlugin : Plugin<Project> {
                 maybeCreate("androidMain").dependsOn(getByName("jvmCommonMain"))
                 maybeCreate("androidUnitTest").dependsOn(getByName("jvmCommonTest"))
 
-                maybeCreate("jsMain").dependsOn(getByName("jvmJsCommonMain"))
-                maybeCreate("jsTest").dependsOn(getByName("jvmJsCommonTest"))
+                maybeCreate("jsMain").apply {
+                    dependsOn(getByName("jvmJsCommonMain"))
+                    dependsOn(getByName("jsWasmJsCommonMain"))
+                }
+                maybeCreate("jsTest").apply {
+                    dependsOn(getByName("jvmJsCommonTest"))
+                    dependsOn(getByName("jsWasmJsCommonTest"))
+                }
+
+                maybeCreate("wasmJsMain").apply {
+                    dependsOn(getByName("jvmJsCommonMain"))
```

Comment:
not sure if I should rename `jvmJsCommon` sourceSets to `jvmJsWasmJsCommon` ones

Reply:
Let's keep it as it is.